src/components/PrefixedEthHashInfo/index.tsx

```diff
@@ -12,7 +13,7 @@ const PrefixedEthHashInfo = ({ hash, ...rest }: Props): ReactElement => {
 
   return (
     <EthHashInfo
-      hash={hash}
+      hash={getAddressWithoutNetworkPrefix(hash)}
```

Comment:
Why are you extracting it here and not sending them via their respective props?

Reply:
I opted to implement it in the wrapper as a _input sanitization_. I don't think it invalidates the correct use of the `hash` prop.

Doing this in the wrapper is also a way to address it in one single place and to not repeat it every time we call the  `<PrefixedEthHashInfo />` component.

Let me know if it makes sense. It is a simple tweak otherwise.

Reply:
I think it is cleaner and more understandable to parse the hash directly and output the props accordingly, i.e.

```ts
  const getPrefixedHashInfoProps = (hash: string, shouldShowShortName: boolean ): { shortName: string | undefined, hash: string }

  <Component
    { ...fn(x, y) }
   .../>
```

Reply:
I am not sure how this improves the solution. My view is that `shortName` should be calculated independently of the `hash` and vice versa.

The way to go would be maybe update the `<EthHashInfo />` to have the input sanitization as the component responsability.

Reply:
@iamacook created a issue to tackle it https://github.com/gnosis/safe-react-components/issues/171